onos-config/values.yaml

```diff
@@ -25,16 +25,30 @@ debug: false
 devices: []
 
 plugins:
-  devicesim:
-    versions:
-      - 1.0.0
-  stratum:
-    versions:
-      - 1.0.0
-  testdevice:
-    versions:
-      - 1.0.0
-      - 2.0.0
+  - name: devicesim
+    version: 1.0.0
+    image:
+      repository: onosproject/config-model-devicesim-1.0.0
+      tag: v0.6.6
+      pullPolicy: Always
```

Comment:
why always? should be same as onos-config?

Reply:
I copied this from a chat stream the Jordan and I had a couple of weeks ago. I'll make it IfNotPresent